src/community.c

```diff
@@ -129,8 +129,8 @@ static int igraph_i_community_eb_get_merges2(const igraph_t *graph,
 
     for (i = igraph_vector_size(edges) - 1; i >= 0; i--) {
         long int edge = (long int) VECTOR(*edges)[i];
-        long int from = IGRAPH_FROM(graph, edge);
-        long int to = IGRAPH_TO(graph, edge);
+        long int from = IGRAPH_FROM(graph, (igraph_integer_t) edge);
```

Comment:
These casts to `igraph_integer_t` are also unnecessary I think?

Reply:
Yes, they are, `IGRAPH_FROM()` and `IGRAPH_TO()` cast their arguments internally.

Reply:
What should I do, shall I correct these in a new PR?

Reply:
Yes, please go ahead if you have the perseverance to do so :)

Reply:
Well, it's a bit waste of time to look at it and then do nothing with it :)